### PR TITLE
📦 NEW: Add automated latest solution stack by regex feature

### PR DESCRIPTION
**Pull request by caiovfernandes**
## Types of changes

What types of changes does your code introduce to <repo_name>?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the CONTRIBUTING.md doc.
- [x] I have added necessary documentation (if appropriate).
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by mcosta1105 — APPROVED** [review]

**Review by luiz-dnx — DISMISSED** [review]
LGTM

**Comment by caiovfernandes** [comment]
@mcosta1105 @luiz-dnx could you guys approve that again?

**Review by luiz-dnx — APPROVED** [review]

File: eb.tf [hunk 1]
```diff
@@ -706,3 +706,9 @@ resource "aws_elastic_beanstalk_environment" "env" {
   }
 }
 
+data "aws_elastic_beanstalk_solution_stack" "solution_stack" {
+  count       = var.solutions_stack_name_regex ? 1 : 0
```
Comment (luiz-dnx) [inline eb.tf:710]: `solutions_stack_name_regex` is a string, it's better to do a conditional based on its length or value (i.e. `try(var.solutions_stack_name_regex, "") != "" ? 1 : 0`.

**Review by luiz-dnx — CHANGES_REQUESTED** [review]

File: eb.tf (hunk 1 above)
Comment (caiovfernandes) [reply eb.tf]: Nice one @luiz-dnx , i didn't saw that.
Fixed now.

**Review by mcosta1105 — APPROVED** [review]

**Review by luiz-dnx — APPROVED** [review]

**Pull request merged**